resources/authgear/templates/en/web/authflowv2/enter_password.html

```diff
@@ -1,122 +1,68 @@
+  {{ template "authflowv2/__password_field.html" (dict
+    "Name" "q_password"
+    "Type" "old-password"
+    "AutoFocus" $.ShouldFocusInput
+    "Classname" "w-full"
+    "Error" .Error
+  ) }}
+{{ if $.DeviceTokenEnabled }}
+  <div class="flex flex-row items-center self-start gap-x-2">
+    <!-- TODO: Integrate checkbox styling from https://github.com/authgear/authgear-server/pull/3648/files#diff-ae1ceba718770ef976b399cb2ca490ae63847974ff7ecf1002c26486e9ce77b5 -->
+    <input class="device-token-input" id="device-token" type="checkbox" name="x_device_token" value="true">
+    <label class="body-text--sm" for="device-token">{{ template "v2-create-device-token-label" }}</label>
   </div>
-  <p class="body-text--sm">
-    By registering, you agree to the <a href="#">Terms of Service</a> and
-    <a href="#">Privacy Policy</a>.
-  </p>
-  <input class="input w-full" placeholder="placeholder"/>
-  <input class="input w-full input--error" invalid placeholder="placeholder"/>
-  <p class="input__error-message mt-2">
-    Please include '@' in the email address. 'Marianne' does not have '@'.
-  </p>
-  {{ template "authflowv2/__alert_message.html"
-     (dict
-      "Type" "error"
-      "Message" "Verification code sending failed. Please try again. Verification code sending failed. Please try again. Verification code sending failed. Please try again."
-    )
-  }}
-
-  <!-- OTP Input -->
-  {{ template "authflowv2/__otp_input.html"
-    (dict
-      "CSRFField" $.CSRFField
-      "CodeLength" 6
-      "ResendButtonLabel" "Resend code"
-      "ResendButtonLabelWithValue" "Resend code (%ds)"
-      "ResendButtonCooldown" 5
-      "SubmitEvent" "authgear.button.submit_verification_code"
-      "ResendEvent" "authgear.button.resend_verification_code"
-    )
-  }}
-
-  <!-- OTP Input - Error -->
-  {{ template "authflowv2/__otp_input.html"
-    (dict
-      "CSRFField" $.CSRFField
-      "CodeLength" 6
-      "ErrorMessage" "Invalid code"
-      "ResendButtonLabel" "Resend code"
-      "ResendButtonLabelWithValue" "Resend code (%ds)"
-      "ResendButtonCooldown" 5
-      "SubmitEvent" "authgear.button.submit_verification_code"
-      "ResendEvent" "authgear.button.resend_verification_code"
-    )
-  }}
-
-  <form data-controller="password-policy">
-    {{ template "authflowv2/__password_field.html" (dict "Type" "new-password" "HasError" "true" "ErrorMessage" "Invalid Password" "Classname" "mt-4") }}
-    {{ template "authflowv2/__password_field.html" (dict "Type" "confirm-password" "Classname" "mt-4") }}
+{{ end }}
+  <button
+    class="primary-btn w-full"
+    type="submit"
+    name="x_action"
+    value=""
+    data-authgear-event="authgear.button.enter_password"
+    data-action-button
+  >
+  <span class="primary-btn__content">
+    {{ template "v2-button.continue" }}
+  </span>
+  <span class="primary-btn__loading">
+    <span class="primary-btn__loading-icon material-icons animate-spin">
+      progress_activity
+    </span>
+  </span>
```

Comment:
We should create a controller to handle the loading state of the button.
We should modify `XHRSubmitFormController` so that we can distinguish a normal disable and a loading disable of the button, probably by setting some additional attributes.
And the new controller should read that attribute, and display the spinner when needed.
We can created another issue for this because it affects many pages.